src/vit-testing/mainnet-tools/src/cardano_cli/fake.rs

```diff
@@ -4,30 +4,44 @@ use serde::{Deserialize, Serialize};
 use serde_json::{json, Value};
 use std::fmt;
 
+/// Arbitrary blockchain tip
+#[must_use]
 pub fn tip() -> Tip {
-    Default::default()
+    Tip::default()
 }
 
+/// Arbitraty query utxo response
+#[must_use]
 pub fn utxo() -> QueryUTxO {
-    Default::default()
+    QueryUTxO::default()
 }
 
+/// Arbitrary hash
+#[must_use]
 pub fn hash() -> String {
     "a3a93043d015e9bb089b1a90d59b1922dffb9684b5c64a61426b6134e348123d".to_string()
 }
 
+/// Arbitrary address
+#[must_use]
 pub fn address() -> String {
     "addr1q9e0wsxghc395p8e3ff0zx2gdzurveq0pcu68lyq2z59vc5squl6qr0re0pe2x5syq7j5qf77q6s7zl43nass8u85vgscexsev".to_string()
 }
 
+/// Arbitrary stake address
+#[must_use]
 pub fn stake_address() -> String {
     "stake1uxgqw0aqph3uhsu4r2gzq0f2qyl0qdg0p06ce7cgr7r6xygwmtwku".to_string()
 }
 
+/// Arbitrary Response from Cardano CLI on submit transaction command
+#[must_use]
 pub fn submit() -> String {
     "Transaction successfully submitted.".to_string()
 }
 
+/// Response from transaction sign command
+#[must_use]
 pub fn sign() -> String {
```

Comment:
could this use the `json!()` macro or a raw string literal to make it a bit more readable?